nodebuilder/das/config.go

```diff
@@ -0,0 +1,70 @@
+package das
+
+import (
+	"errors"
+	"time"
+)
+
+type Config struct {
+	//  samplingRange is the maximum amount of headers processed in one job.
+	SamplingRange uint
+
+	// concurrencyLimit defines the maximum amount of sampling workers running in parallel.
+	ConcurrencyLimit uint
+
+	// backgroundStoreInterval is the period of time for background checkpointStore to perform a checkpoint backup.
+	BackgroundStoreInterval time.Duration
+
+	// priorityQueueSize defines the size limit of the priority queue
+	PriorityQueueSize uint
+
+	// genesisHeight is the height sampling will start from
+	GenesisHeight uint
+}
+
+// TODO(@derrandz): parameters needs performance testing on real network to define optimal values
+func DefaultConfig() Config {
```

Comment:
note: @derrandz, this is where `node.Type` will be passed eventually to have different Configs for each type

Reply:
Noted.

Reply:
Keeping track of this in #1261